packages/cozy-stack-client/src/KonnectorCollection.js

```diff
@@ -0,0 +1,64 @@
+import { normalizeDoc } from './DocumentCollection'
+import { uri } from './utils'
+
+export const KONNECTORS_DOCTYPE = 'io.cozy.konnectors'
+
+export const normalizeApp = konnector => {
+  return {
+    ...konnector,
+    ...normalizeDoc(konnector, KONNECTORS_DOCTYPE),
+    ...konnector.attributes
+  }
+}
+
+/**
+ * Implements its own methods specific to io.cozy.konnectors
+ */
+class KonnectorCollection {
+  constructor(stackClient) {
+    this.stackClient = stackClient
+  }
+
+  /**
+   * Lists all konnectors, without filters.
+   *
+   * The returned documents are not paginated by the stack.
+   *
+   * @return {{data, meta, skip, next}} The JSON API conformant response.
+   * @throws {FetchError}
+   */
+  async all() {
+    const path = uri`/konnectors/`
+    const resp = await this.stackClient.fetchJSON('GET', path)
+    return {
+      data: resp.data.map(konnector => normalizeApp(konnector)),
+      meta: {
+        count: resp.meta.count
+      },
+      skip: 0,
+      next: false
+    }
```

Comment:
I think AppCollection and KonnectorCollection are very similar ? Why don't they share code ?

Reply:
You're right, let met try something to use the same codebase here 👍